specification/ai/HealthInsights/HealthInsights.Common/model.common.shared.tsp

```diff
@@ -1,7 +1,49 @@
+using TypeSpec.Http;
 using TypeSpec.Rest;
+using Azure.Core;
 
 namespace AzureHealthInsights;
 
+/**
+ * Provides HealthInsights resource details for long running operations.
+ * @template TRequest The type of the operation request data.
+ * @template TResult The type of the operation result.
+ * @template TStatusError The type of the operation status error. If not provided, the default error is used.
+ */
+// @TypeSpec.Versioning.added(AzureHealthInsights.ApiVersion.v2024_04_01)
+@doc("Provides status details for long running operations.")
+model HealthInsightsJobBase {
+  @key("id")
+  @doc("The unique ID of the job.")
+  @visibility("read")
+  @maxLength(36)
+  @minLength(3)
+  @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_]+[a-zA-Z0-9]$")
+  @path
```

Comment:
Recommend remove `@path` here to avoid separate CreateOrUpdate schema.
```suggestion
```

Reply:
Resolved!.
Thanks